src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/ResponsibilityTests.kt

```diff
@@ -75,11 +75,7 @@ class ResponsibilityTests : DatabaseTest()
     fun `can get touchstones by group id`()
     {
         validate("/modelling-groups/$groupId/responsibilities/") against "Touchstones" given {
-            it.addGroup(groupId)
-            it.addTouchstoneName("touchstone", "description")
-            it.addTouchstone("touchstone", 1, description = "descr 1", status = "open")
-            it.addTouchstone("touchstone", 2)
-            it.addResponsibilitySet(groupId, touchstoneId)
```

Comment:
we already have a function in this class that sets up the data as required, including open responsibilites